beast/fitting/fit.py

```diff
@@ -262,7 +264,7 @@ def Q_all_memory(prev_result, obs, sedgrid, ast, qnames_in, p=[16., 50., 84.],
 
     # get the names of all the children in the ast structure
     ast_children = []
-    for label, node in ast.root._v_children.items(): 
+    for label, node in list(ast.root._v_children.items()): 
```

Comment:
Are we mixing h5py and pytables? if so we should only use one.
(list is not needed)